cli/ioctl/cmd/action/actionhash.go

```diff
@@ -114,7 +114,15 @@ func printAction(actionInfo *iotexapi.ActionInfo) (string, error) {
 	output += fmt.Sprintf("actHash: %s\n", actionInfo.ActHash)
 	return output, nil
 }
-
+func toIOTX(amount string) (iotx string, err error) {
+	amountInt, err := util.StringToRau(amount, 0)
```

Comment:
This is confusing. Can you please implement StringToIOTX() in util/util.go?

Reply:
sure